src/Microsoft.AspNetCore.BuildTools.ApiCheck/ApiListingComparer.cs

```diff
@@ -76,6 +76,24 @@ private void CompareMembers(TypeDescriptor type, TypeDescriptor newType, List<Ap
                     }
                 }
             }
+
+            if (type.Kind == TypeKind.Interface && type.Members.Count < newType.Members.Count)
```

Comment:
Gotcha. Okies. I'm not super familiar with how the code here is structured so if it looks good to you :shipit: